frontend/src/components/SearchImage/SearchImage.jsx

```diff
@@ -1,28 +1,49 @@
-import React, { useRef } from 'react'
-import './SearchImage.scss';
+import React, { useRef, useState, useEffect } from "react";
+import "./SearchImage.scss";
 
 const SearchImage = ({ onChange, setInputValue, value }) => {
-    const searchRef = useRef();
+  const searchRef = useRef();
+  const [isSearchBarFull, setSearchBarFull] = useState(true);
```

Comment:
```suggestion
  const [isSearchBarFull, setSearchBarFull] = useState(false);
```
I think default value for `isSearchBarFull` should be `false`

Reply:
resolve